keyboards/sawnsprojects/krush/krush65/hotswap/info.json

```
@@ -0,0 +1,157 @@
+{
+    "keyboard_name": "Krush65 Hotswap",
+    "url": "",
+    "maintainer": "Mai The San",
```

This field is meant to reference your GitHub username.

```suggestion
    "maintainer": "MaiTheSan",
```